### PR TITLE
Refactor more widget definitions

### PR DESCRIPTION
**Pull request by AaronVanGeffen**
Part of #12329

* Banner
* Changelog
* Custom currency
* Debug paint
* Demolish and refurbish ride prompts
* Finances (summary only)
* Quit prompt

### CODE REVIEW

File: src/openrct2-ui/windows/Changelog.cpp [hunk 1]
```diff
@@ -40,8 +40,8 @@ constexpr int32_t MIN_WH = 200;
 
 static rct_widget window_changelog_widgets[] = {
     WINDOW_SHIM(WINDOW_TITLE, WW, WH),
-    { WWT_RESIZE,           1,  0,          WW - 1, 14,     WH - 1,     0xFFFFFFFF,                     STR_NONE },             // content panel
-    { WWT_SCROLL,           1,  3,          WW - 3, 16,     WH - 15,    SCROLL_BOTH,                    STR_NONE },             // scroll area
+    MakeWidget({0, 14}, {500, 386}, WWT_RESIZE, 1, 0xFFFFFFFF,  STR_NONE), // content panel
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Changelog.cpp:43]:
```suggestion
    MakeWidget({0, 14}, {500, 386}, WWT_RESIZE, 1                       ), // content panel
```

Comment (AaronVanGeffen) [reply src/openrct2-ui/windows/Changelog.cpp:43]: Nice catch!

File: src/openrct2-ui/windows/Changelog.cpp [hunk 2]
```diff
@@ -40,8 +40,8 @@ constexpr int32_t MIN_WH = 200;
 
 static rct_widget window_changelog_widgets[] = {
     WINDOW_SHIM(WINDOW_TITLE, WW, WH),
-    { WWT_RESIZE,           1,  0,          WW - 1, 14,     WH - 1,     0xFFFFFFFF,                     STR_NONE },             // content panel
-    { WWT_SCROLL,           1,  3,          WW - 3, 16,     WH - 15,    SCROLL_BOTH,                    STR_NONE },             // scroll area
+    MakeWidget({0, 14}, {500, 386}, WWT_RESIZE, 1                       ), // content panel
+    MakeWidget({3, 16}, {495, 370}, WWT_SCROLL, 1, SCROLL_BOTH, STR_NONE), // scroll area
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Changelog.cpp:44]:
```suggestion
    MakeWidget({3, 16}, {495, 370}, WWT_SCROLL, 1, SCROLL_BOTH          ), // scroll area
```

File: src/openrct2-ui/windows/CustomCurrency.cpp [hunk 3]
```diff
@@ -35,10 +35,10 @@ enum WINDOW_CUSTOM_CURRENCY_WIDGET_IDX {
 
 static rct_widget window_custom_currency_widgets[] = {
     WINDOW_SHIM(WINDOW_TITLE, WW, WH),
-      SPINNER_WIDGETS      (1,  100,    200,    30,     40, STR_CHEAT_CURRENCY_FORMAT,          STR_NONE), // NB: 3 widgets
-    { WWT_BUTTON,           1,  120,    200,    50,     60, 0,                                  STR_NONE },
-    { WWT_DROPDOWN,         1,  220,    350,    50,     60, STR_NONE,                           STR_NONE },
-    { WWT_BUTTON,           1,  339,    349,    51,     59, STR_DROPDOWN_GLYPH,                 STR_NONE },
+    MakeSpinnerWidgets({100, 30}, {101, 11}, WWT_SPINNER,  1, STR_CHEAT_CURRENCY_FORMAT), // NB: 3 widgets
+    MakeWidget        ({120, 50}, { 81, 11}, WWT_BUTTON,   1, STR_NONE                 ),
```
Comment (AaronVanGeffen) [inline src/openrct2-ui/windows/CustomCurrency.cpp:39]: The original reads `0`; this should be `STR_EMPTY`, not `STR_NONE`.
```suggestion
    MakeWidget        ({120, 50}, { 81, 11}, WWT_BUTTON,   1, STR_EMPTY                ),
```

File: src/openrct2-ui/windows/Changelog.cpp [hunk 4]
```diff
@@ -40,8 +40,8 @@ constexpr int32_t MIN_WH = 200;
 
 static rct_widget window_changelog_widgets[] = {
     WINDOW_SHIM(WINDOW_TITLE, WW, WH),
-    { WWT_RESIZE,           1,  0,          WW - 1, 14,     WH - 1,     0xFFFFFFFF,                     STR_NONE },             // content panel
-    { WWT_SCROLL,           1,  3,          WW - 3, 16,     WH - 15,    SCROLL_BOTH,                    STR_NONE },             // scroll area
+    MakeWidget({0, 14}, {500, 386}, WWT_RESIZE, 1                       ), // content panel
+    MakeWidget({3, 16}, {495, 370}, WWT_SCROLL, 1, SCROLL_BOTH          ), // scroll area
```
Comment (Gymnasiast) [inline src/openrct2-ui/windows/Changelog.cpp:44]: Now both definitions have been shortened, there is excessive whitespace here.

**Comment by Gymnasiast** [comment]
Mind the dot after the commit message.

**Pull request merged**